src/zhinst/toolkit/driver/modules/shfqa_sweeper.py

```diff
@@ -297,3 +295,12 @@ def get_offset_freq_vector(self) -> t.Any:
         """
         self._update_settings()
         return self._raw_module.get_offset_freq_vector()
+
+    @staticmethod
+    def _create_daq(server_host, server_port):
+        try:
+            return ziDAQServer(server_host, server_port, 6, allow_version_mismatch=True)
+        except TypeError as error:
+            if "allow_version_mismatch" not in error.args[0]:
+                raise
+            return ziDAQServer(server_host, server_port, 6)
```

Comment:
suggestion: Instead of duplicating the logic here I would instead create a member function in the `Session` class called `clone_underlying_session` that returns a new session. This would keep the logic concise in one place.